src/test/unit/controls/Gantt/Gantt-spec.js

```diff
@@ -2491,4 +2492,79 @@ describe("Gantt", () => {
             expect(gantt.resizeHandler).toBeNull();
         });
     });
+    describe("When pan is enabled", () => {
```

Comment:
`describe("Panning", () => {`
We are checking stuff in here that are applicable when panning is not enabled as well.

Reply:
I will change the description